scripts/README.rst

```diff
@@ -0,0 +1,145 @@
+Update this document once the above card is complete
+
+By far the most used and important script in this directory. The
+deploy.sh script is used to deploy RPCO with certain features turned on
+or off. These are toggled via environment variables described
+`here <https://github.com/rcbops/rpc-openstack/tree/master#environment-variables-for-deploysh>`__.
+
+When adding a new feature into RPCO, there might be a need to add a
+corresponding DEPLOY variable into this script so the feature can be
+toggled on or off. Please follow the same pattern outlined in this
+script, and enclose any instructions specific to deploying the new
+feature with a conditional similar to what is done
+`here <https://github.com/rcbops/rpc-openstack/blob/master/scripts/deploy.sh#L226>`__
+for ceph.
+
+f5-config.py
+------------
+
+NOTE: The f5-config.py script is currently being reworked as part of
+https://github.com/rcbops/u-suk-dev/issues/836 TODO: (alextricity25)
+Update this section once the above card closes
+
+The f5-config.py script is used to generate f5 load balancer commands to
+create the necessary components on the f5 for RPCO. This includes
+creating the virtual servers, node pools, monitors, and more.
+
+This script is used by the `LEM
+f5\_deploy.sh <https://github.com/rcbops/rpc_lem/blob/master/lem-aide/user-tools/f5_deploy.sh>`__
+script to assist deployers with automatically configuring the f5 in the
+LEM labs.
+
+To add a service to the f5-config.py script, an entry must be added to
+the
+`MONITORS <https://github.com/rcbops/rpc-openstack/blob/master/scripts/f5-config.py#L54>`__
+list and
+`POOL\_PARTS <https://github.com/rcbops/rpc-openstack/blob/master/scripts/f5-config.py#L173>`__
+dictionary for each service that your adding.
+
+f5-monitor.sh
+-------------
+
+Used to build MaaS remote checks for the external endpoints on the f5
+
+functions.sh
+------------
+
+Functions that are used by the deployment scripts(deploy.sh or
+deploy-rpc-playbook.sh). This file is sourced to bring in the functions
+implemented in this file.
+
+linting-ansible.sh
+------------------
+
+Is used by our tox tests, which are run by the travis ci, to check the
+syntax and perform linting checks on the playbooks in RPCO. This script
+is invoked
+`here <https://github.com/rcbops/rpc-openstack/blob/master/tox.ini#L50>`__
+
+linting-bashate.sh
+------------------
+
+Is used by our tox tests, which are run by the travis ci, to check the
+syntax and peform linting checks on the bash scripts used in RPCO. This
+script is invoked
+`here <https://github.com/rcbops/rpc-openstack/blob/master/tox.ini#L43>`__
+
+linting-pep8.sh
+---------------
+
+Is used by our tox tests, which are run by the travis ci, to check the
+python scripts used in RPCO are pep8 compliant. This script is invoked
+`here <https://github.com/rcbops/rpc-openstack/blob/master/tox.ini#L39>`__
+
+migrate-yaml.py
+---------------
+
+TODO: (alextricity25) Remove this in the master and newton-14.0 branches
```

Comment:
Good point. I'm going to bring this up in https://github.com/rcbops/u-suk-dev/issues/802 as it seems fitting for that card.